src/commands/goto.ts

```diff
@@ -25,43 +25,44 @@ const jumps: [string, string][] = [
 function executeGoto(gotoType: number, editor: vscode.TextEditor, count: number, extend: boolean) {
```

Comment:
- I don't think you ever return a `Promise` here, so I'm wondering why you changed all the functions in order to return `Promise.resolve()` everywhere?
- In the `switch`, we don't need the `break` statements since we have `return`s before all of them.